pkg/storage/metrics.go

```diff
@@ -155,6 +155,9 @@ var (
 	metaSysCount = metric.Metadata{
 		Name: "syscount",
 		Help: "Count of system KV pairs"}
+	metaWritesPerSecond = metric.Metadata{
+		Name: "writes_per_second",
```

Comment:
ah, then yes, the exact value used by the rebalancing makes sense.
But then it really needs a better name. eg: `rebalancer.estimated_writes_per_second` or something like that.